internal/pkg/task/config_runner.go

```diff
@@ -65,14 +71,21 @@ func (r *ConfigRunner) Run() ([]*Task, error) {
 		}
 		r.Subnets = subnets
 	}
+	platformVersion := "1.4.0"
```

Comment:
I wonder if we can leave this empty and only set `1.0.0` if it's window and the command will work? 🤔 
This way if LATEST ever changes we don't need to update it here. (or we can set it to LATEST)

Reply:
I believe it has been hard-coded as `1.4.0` in the past out of concern that if there's a new LATEST that breaks things, we'll be in a bad way, so it's better to manually change it to a future platform version once we've confirmed that all is fine. But yeah, it is an optional field so if we decide we're comfortable with LATEST, then we should be able to leave it empty.

Reply:
Personally, I think it makes sense for us to stick to LATEST for the PlatformVersion, and then switch it up if we detect certain operating systems/architectures

Reply:
@iamhopaul123 How do you feel? (You were the one who questioned it in the design doc.)

Reply:
Discussed offline. Changed "1.4.0" for PlatformVersion to "LATEST" throughout codebase.

Reply:
Sounds good to me!